### DISCUSSION TITLE
Slugs in front-end site have blank values

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#1607] [Q&A] [answered]
I have a site that was made in Laravel and later I added twill to it.   I'm therefore referencing the front-end website model for a data object I have. 

It seems that the model isn't linked to the model_slugs table as when using eloquent to loop through results, if I query the slug value I get a blank result. 

To give you an example:

I have a table called 'clusters' and a related table called 'cluster_slugs'.    Here is a simple test eloquent query:

```
$clusters=\App\Models\Cluster::active()->where("a","=",$b)->where("c","!=",d)->get();
foreach ($clusters as $cluster){
    echo $cluster->title;    // works fine
    echo $cluster->slug;    // doesn't work
}
```

This is something very simple I am sure about connecting the tables ?

**Comment by haringsrob** [comment, marked as answer]
Hey @arkid,

Is your model correctly extending the twill model class?

**Reply by adam-jones-net** [reply]
It is now, thanks @haringsrob
